CMake/External_PyBind11.cmake

```diff
@@ -0,0 +1,27 @@
+if (NOT fletch_BUILD_CXX11)
+  message(FATAL_ERROR "CXX11 must be enabled to use PyBind11")
+endif()
+
+ExternalProject_Add(PyBind11
+  URL ${PyBind11_url}
+  URL_MD5 ${PyBind11_md5}
+  PREFIX ${fletch_BUILD_PREFIX}
+  DOWNLOAD_DIR ${fletch_DOWNLOAD_DIR}
+  INSTALL_DIR ${fletch_BUILD_INSTALL_PREFIX}
+  CMAKE_GENERATOR ${gen}
+  CMAKE_ARGS
+    ${COMMON_CMAKE_ARGS}
+    -DPYBIND11_TEST:BOOL=OFF # To remove dependencies; build can still be tested manually
+  )
+
+fletch_external_project_force_install(PACKAGE PyBind11)
+
+set(PyBind11_ROOT "${fletch_BUILD_INSTALL_PREFIX}" CACHE PATH "" FORCE)
+file(APPEND ${fletch_CONFIG_INPUT} "
+################################
+# PyBind11
+################################
+set(PyBind11_ROOT @PyBind11_ROOT@)
```

Comment:
The pattern here has changed. All 'ROOT' variables are now relative to Fletch_ROOT. You can see one of the other External files for the template.